priority-queue/Readme.md

```diff
@@ -4,43 +4,71 @@ This document describes the Priority Queue Pattern example from the guide [Cloud
 
 ## System Requirements
 
-* Microsoft .NET Framework version 4.5
-* Microsoft Visual Studio 2015 Community, Enterprise, or Professional
-* Windows Azure SDK for .NET version 2.9
+* Microsoft .NET Core 3.1
+* Microsoft Visual Studio 2019
+* Windows Azure SDK for .NET version 3.1
 
 ## Before you start
 
 Ensure that you have installed all of the software prerequisites.
 
-The example demonstrates operational aspects of applications running in Windows Azure. Therefore, you will need to use the diagnostics tools in order to understand how the code sample works. You **must** ensure that the web and worker roles in the solution are configured to use the diagnostics mechanism. If not, you will not see the trace information generated by the example.
+The example demonstrates operational aspects of applications running in Windows Azure. Therefore, you will need to use the monitoring tools in order to understand how the code sample works. You **must** ensure that the Azure Functions in the solution are configured to use the diagnostics mechanism. If not, you will not see the trace information generated by the example.
 
 ## About the Example
 
-This example shows how you can implement priority queues by using Service Bus Topics and Subscriptions. A worker role is responsible for sending messages to a topic. It assigns a priority to each message. The receiving worker roles read messages from subscriptions that have the corresponding priority. In the example, The PriorityQueue.High worker role runs with two instances, whereas the PriorityQueue.Low worker runs only with one. This ensures that high priority messages are read from the queue more quickly than low priority messages.
-
+This example shows how you can implement priority queues by using Service Bus Topics and Subscriptions. A timer triggered Azure Function is responsible for sending messages to a topic. It assigns a priority to each message. The receiving Azure Functions read messages from subscriptions that have the corresponding priority. In the example, The PriorityQueueConsumerHigh Azure function can scale out to 200 instances, whereas the PriorityQueueConsumerLow worker runs only with one instance. This ensures that high priority messages are read from the queue more quickly than low priority messages.
 
 ## Running the Example
 
-You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure Cloud Service.
+You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Function App Service.
```

Comment:
Is the Visual Studio Windows Azure emulator really still necessary?  Maybe it is?

Reply:
I'm resolving this as I've updated this part in the readme on my recent review.